test/service/test_response.py

```diff
@@ -1261,6 +1270,98 @@ def test_bundles_outer_entity(self):
+                                "gte": 2049840000.0,
+                                "lte": 2175984000.0
+                            },
+                            {
+                                "gte": 1419120000.0,
+                                "lte": 1545264000.0
+                            }
+                        ],
+                        "organismAgeUnit": [
+                            "year"
+                        ]
+                    }
+                ],
+                [
+                    {
+                        "biologicalSex": [
+                            "male",
+                            "female"
+                        ],
+                        "disease": None,
+                        "genusSpecies": [
+                            "Homo sapiens"
+                        ],
+                        "id": [
+                            "HPSI0314i-hoik",
+                            "HPSI0214i-wibj",
+                            "HPSI0314i-sojd",
+                            "HPSI0214i-kucg"
+                        ],
+                        "organismAge": [
+                            "40-44",
+                            "55-59"
+                        ],
+                        "organismAgeRange": [
+                            {
+                                "gte": 1734480000.0,
+                                "lte": 1860624000.0
+                            },
+                            {
+                                "gte": 1261440000.0,
+                                "lte": 1387584000.0
+                            }
+                        ],
+                        "organismAgeUnit": [
+                            "year"
+                        ]
+                    }
+                ]
+            ]
+
+        url = self.base_url + '/repository/projects'
+        for relation, range_value, expected_hits in [('contains', (1419130000, 1545263000), test_hits[:1]),
+                                                     ('within', (1261430000, 1545265000), test_hits),
+                                                     ('intersects', (1860623000, 1900000000), test_hits[1:]),
+                                                     ('contains', (1860624000, 2049641000), []),
+                                                     ('within', (1734490000, 1860623000), []),
+                                                     ('intersects', (1860624100, 2049641000), [])]:
+
+            with self.subTest(relation=relation, value=range_value):
+                params = {
+                    'filters': json.dumps({'organismAgeRange': {relation: [range_value]}}),
+                    'order': 'desc',
+                    'sort': 'entryId'
+                }
+            response = requests.get(url, params=params)
+            actual_value = [hit['donorOrganisms'] for hit in response.json()['hits']]
+            self.assertElasticsearchResultsEqual(expected_hits, actual_value)
```

Comment:
These three lines need to be indented so they are part of the body of the subTest context manager.